.github/workflows/docker-nightly.yaml

```diff
@@ -0,0 +1,34 @@
+name: Nightly test
+on:
+  schedule:
+    # Execute at 2am EST every day
+    - cron:  '0 7 * * *'
+
+jobs:
+  vars:
+    runs-on: ubuntu-latest
+    outputs:
+      REGISTRY_NAME: "k8scc01covidacr.azurecr.io"
```

Comment:
Same as the other comment about registry name.

Reply:
For nightly, everything is static, so that can be changed to env to clean it up a bit.